config/node.go

```diff
@@ -2,33 +2,27 @@ package config
 )
 
 type Node struct {
-	ListenAddrs               []string              `json:"listen_addrs"`
-	AnnounceAddrs             []string              `json:"announce_addrs"`
-	PeeringPeers              []peering.PeeringPeer `json:"peering_peers"`
-	IndexerTickInterval       int                   `json:"indexer_tick_interval"`
-	EnableWebsocketListenAddr bool                  `json:"enable_websocket_listen_addr"`
-	HardFlushWriteLog         bool                  `json:"hard_flush_write_log"`
-	WriteLogTruncate          bool                  `json:"write_log_truncate"`
-	NoBlockstoreCache         bool                  `json:"no_blockstore_cache"`
-	NoLimiter                 bool                  `json:"no_limiter"`
-	IndexerURL                string                `json:"indexer_url"`
-	Blockstore                string                `json:"blockstore"`
-	WriteLogDir               string                `json:"write_log_dir"`
-	Libp2pKeyFile             string                `json:"libp2p_key_file"`
-	DatastoreDir              string                `json:"datastore_dir"`
-	WalletDir                 string                `json:"wallet_dir"`
-	ApiURL                    string                `json:"api_url"`
-	Bitswap                   Bitswap               `json:"bitswap"`
-	Limits                    Limits                `json:"limits"`
-	ConnectionManager         ConnectionManager     `json:"connection_manager"`
-}
-
-func (cfg *Node) GetLimiter() *rcmgr.BasicLimiter {
-	lim := rcmgr.NewDefaultLimiter()
-	cfg.Limits.apply(lim)
-	return lim
+	ListenAddrs               []string                 `json:"listen_addrs"`
+	AnnounceAddrs             []string                 `json:"announce_addrs"`
+	PeeringPeers              []peering.PeeringPeer    `json:"peering_peers"`
+	IndexerTickInterval       int                      `json:"indexer_tick_interval"`
+	EnableWebsocketListenAddr bool                     `json:"enable_websocket_listen_addr"`
+	HardFlushWriteLog         bool                     `json:"hard_flush_write_log"`
+	WriteLogTruncate          bool                     `json:"write_log_truncate"`
+	NoBlockstoreCache         bool                     `json:"no_blockstore_cache"`
+	NoLimiter                 bool                     `json:"no_limiter"`
+	IndexerURL                string                   `json:"indexer_url"`
+	Blockstore                string                   `json:"blockstore"`
+	WriteLogDir               string                   `json:"write_log_dir"`
+	Libp2pKeyFile             string                   `json:"libp2p_key_file"`
+	DatastoreDir              string                   `json:"datastore_dir"`
+	WalletDir                 string                   `json:"wallet_dir"`
+	ApiURL                    string                   `json:"api_url"`
+	Bitswap                   Bitswap                  `json:"bitswap"`
+	Limits                    rcmgr.ScalingLimitConfig `json:"limits"`
```

Comment:
this is a break to the json structure of node config, if NoLimiter is set to false. from a brief scan it looks like this is hardcoded to true, so the limits codepath doesn't even get executed currently. but if we ever switch this back to true we'd need to ensure the `limits` json blob matches the `ScalingLimitConfig` struct rather than the custom `Limits` struct we had previously - which no longer maps cleanly into the `ScalingLimitConfig` which is why i removed it entirely.